packages/specs/ajv/src/services/AjvService.ts

```
@@ -23,7 +24,13 @@ export class AjvService {
   protected ajv = inject(Ajv);
 
   async validate(value: any, options: AjvValidateOptions | JsonSchema): Promise<any> {
-    let {schema: defaultSchema, type, collectionType, ...additionalOptions} = this.mapOptions(options);
+    let {
```

Biome (1.9.4)

[error] This let declares some variables that are only assigned once.

'defaultSchema' is never reassigned.

'type' is never reassigned.

'collectionType' is never reassigned.

'returnsCoercedValues' is never reassigned.

'additionalOptions' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)